src/js/rx/components/MessageProviderLink.jsx

```diff
@@ -2,8 +2,14 @@ import React from 'react';
 
 class MessageProviderLink extends React.Component {
   render() {
-    // TODO: Link to secure messaging.
-    return <a className="usa-button rx-message-provider-link">Message my provider</a>;
+    return (
+      <a
+          className="usa-button rx-message-provider-link"
+          href="/messaging"
+          target="_blank">
```

Comment:
I used `_blank` for target since that's the behavior @gnakm specified. https://dsva.slack.com/archives/vetsgov-prescriptions/p1476464059000372